### ISSUE TITLE
🐛 No Acronym definitions for VoiceOver

### ISSUE THREAD

**Issue by BrookeCameron** [GCTC-NTGC/gc-digital-talent#13359]
### 🐛 Bug and Expected Behaviour

There is no way to interact or view the acronym definitions when using voice over. This is important context that visual curser users receive when interacting with our site.

### 🕵️ Details

**Operating system**: Mac OS
**Browser**: Safari 
**Assistive Tech**: VoiceOver

### 📸 Screenshot

https://github.com/user-attachments/assets/48d8521d-6f2f-4d64-89a5-78f86efd1d01

**Comment by petertgiles** [comment]
@esizer we use our `wrapAbbr` helper for this which adds `abbr` tags.  This seems like pretty basic behaviour - can Safari really not access it?  Is there something else we should be doing?

**Comment by esizer** [comment]
> [@esizer](https://github.com/esizer) we use our `wrapAbbr` helper for this which adds `abbr` tags. This seems like pretty basic behaviour - can Safari really not access it? Is there something else we should be doing?

Ahhh, yeah. That is similar to the Sign-In button thing. We use `abbr[title=]` and a lot of ATs ignore the title attribute so we should avoid using it.

**Comment by petertgiles** [comment]
Do you have a recommendation on a better way to handle abbreviations?

**Comment by esizer** [comment]
> Do you have a recommendation on a better way to handle abbreviations?

Not a solution that will be easily accepted or to implement 😅 However, using `abbr` with `title` is probably only useful for sighted users (giving the tooltip from title) so we can't really continue with that assuming it is helping AT users.

The current recommendations are:

1. Define it on first use and then continue using it after that
     - Information Technology (IT)  ... IT
2. Link it to a glossary of definitions and have an explanation there

Honestly, acronyms are just bad for everyone involved. It excludes people who are unaware of them whether they rely on ATs or not so we should be attempting to avoid where possible. These ones are tricky though because I can't think of a way to avoid these ones 🤷‍♀ I doubt people will want to spell out the classification 😢

**Comment by petertgiles** [comment]
Lol, OK.  We also have [an issue to use more abbreviation tags](https://github.com/GCTC-NTGC/gc-digital-talent/issues/12959).  😅 

It sounds like there might be nothing for devs to action in this issue?  Other than passing this request over to @NienkeBr ?

**Comment by esizer** [comment]
> Lol, OK. We also have [an issue to use more abbreviation tags](https://github.com/GCTC-NTGC/gc-digital-talent/issues/12959). 😅
> 
> It sounds like there might be nothing for devs to action in this issue? Other than passing this request over to [@NienkeBr](https://github.com/NienkeBr) ?

Yeah, that ticket is still valuable because the `wrapAbbr` function still has vlaue in that it makes it so screen readers pronounce the abbreviations properly. 

Abbreviations should have periods "I.T", however, ours do not because they are more akin to letter codes that acronyms so we need the `aria-label` to split it up as if they are distinct letters "I T" because otherwise screen readers do their best to announce them like words "it"

**Comment by BrookeCameron** [comment]
@petertgiles should I book a QA Session with a Fable user?
